src/NServiceBus.AcceptanceTests/EndpointTemplates/Requires.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    static class Requires
+    {
+        static readonly TestSuiteConstraints constraints = new TestSuiteConstraints();
```

Comment:
Would it work if this was:

```C#
static readonly ITestSuiteConstraints constraints = new TestSuiteConstraints();
```

...to remove the requirement for the empty partial class?